pkg/policy/api/rule_validation.go

```diff
@@ -557,18 +558,30 @@ func (c CIDR) sanitize() error {
 // valid, and ensuring that all of the exception CIDR prefixes are contained
 // within the allowed CIDR prefix.
 func (c *CIDRRule) sanitize() error {
-
-	// Either CIDRGroupRef or Cidr is allowed
-	if len(c.CIDRGroupRef) == 0 && len(c.Cidr) == 0 {
-		return fmt.Errorf("either cidrGroupRef or cidr are required")
+	// Exactly one of CIDR, CIDRGroupRef, or CIDRGroupSelector must be set
```

Comment:
nit: It might be nice to have the unit test for this function extended to cover the new logic as well